factory.go

```diff
@@ -35,6 +36,8 @@ type LoggerFactory struct {
 	// Format as string, values are "text" or "json", by default "text" is used.
 	// when a terminal is not detected "json" is used instead.
 	Format string
+	// TimeFormat is used for marshaling timestamps
+	TimeFormat string
```

Comment:
This should be removed since shouldn't change, is a company standard.

Reply:
So how you can set up `TimeFormat` in your project (I mean precision). For instance @rporres asked here: https://github.com/src-d/borges/issues/254 for _microsecond precision_ for borges.

Reply:
You can remove this from the the PR since, was done at https://github.com/src-d/go-log/pull/5

Reply:
What I am saying is that the configuration of the formats can't be done per project basis